pkgs/development/haskell-modules/configuration-ghc-9.12.x.nix

```diff
@@ -138,6 +138,23 @@ with haskellLib;
   # https://gitlab.haskell.org/ghc/ghc/-/issues/25930
   generic-lens = dontCheck super.generic-lens;
 
+  hashable =
```

Comment:
Why are these overrides all in the versioned GHC configurations? They don't seem to be version specific and should go into `configuration-common.nix`.

Reply:
Unfamiliarity with Haskell ecosystem stuff. Will move it there!